en/graph-views/secure-your-map-platform.md

```diff
@@ -233,6 +233,19 @@ RUN_ARGS="--spring.profiles.active=prod,tls_broker"
 > "tls_broker" profile implies "tls" profile. So Centreon MAP service
 > serves necessarily HTTPS.
 
+Once you add a truststore, Centreon will only use the truststore to validate certificates. 
+This means that if you use self-signed certificates, you also need to add the central server's certificate to the truststore. If you don't, the
```

Comment:
This means that if you use self-signed certificates for the central server, you have to add it to the truststore.